app/code/Magento/Customer/Model/Customer.php

```diff
@@ -537,7 +545,8 @@ public function getAddresses()
     /**
      * Retrieve all customer attributes
      *
-     * @return Attribute[]
+     * @return array
```

Comment:
It’s better to keep original return statement there. Or it was incorrect?

Reply:
Yes, the origin return statement is correct. 

Thanks, @ihor-sviziev